src/js/plugin/external/PluginHelper.js

```diff
@@ -1,27 +0,0 @@
-import PluginDispatcher from "./PluginDispatcher";
```

Comment:
Let's stop here as I really want to move on with the whole plugin thing. I will reintegrate the PluginHelper and remove the SDK repo to bring this discussion to an end.

Please note that I'm not happy with sharing all these objects globally as I don't see any value but a few downsides. I also do not recommend to have any dependency on a different plugin, so please don't use any MountPoints provided by a plugin as the risk is high that these are `undefined` at the moment your plugin is loaded or even worse the depended plugin is never loaded (e.g. not included in the plugin-conf.json).